lib/discover/global.go

```diff
@@ -46,8 +45,7 @@ const (
 )
 
 type announcement struct {
-	Direct []string `json:"direct"`
-	Relays []Relay  `json:"relays"`
+	Addresses []string `json:"addresses"`
```

Comment:
Do we handle compatibility for this on the discosrv (i.e. craft an answer with all of direct, relays, and addresses fields in it), or do you envision it as a clean break with the old?